manual/manual/refman/exten.etex

```diff
@@ -1792,6 +1792,19 @@ the attributes are considered to apply to the payload:
 fun%foo[@bar] x -> x + 1 === [%foo (fun x -> x + 1)[@bar ] ];
 \end{verbatim}
 
+Quoted strings "{|...|}" are particularly interesting for payloads
+that embed foreign syntax fragments. Those fragments can be interpreted
+by a preprocessor and turned into OCaml code without requiring escaping
+quotes. For instance, you can use "[%sql {|...|}]" to
+represent arbitrary SQL statements -- assuming you have a ppx-rewriter
+that recognizes the "%sql" extension.
+
+Note that the non-extension form, for example "{sql|...|sql}", should
```

Comment:
I would change this to "the sigil-quoted form" or "prefix-quoted form" or some such. After all, none of the `{|` quoted string forms are for extensions, so calling this the "non-extension form" makes it sound like the _other_ form is only for extensions.

Reply:
What do you think of "word-delimited form" ?

Reply:
I like!